### PR TITLE
Installer defaults from environment variables

### LINKED ISSUE #1886: Define installer defaults using environment variables
For example
```sh
NAMELESS_HOSTNAME="example.com"
NAMELESS_FRIENDLY="true"
NAMELESS_DATABASE_USERNAME="root"
NAMELESS_DATABASE_ADDRESS="localhost"
```
https://www.php.net/manual/en/function.getenv.php

This would be nice for docker users since container configuration in docker is done primarily using environment variables

### PR DESCRIPTION
**Pull request by Derkades**
Speeds up installation for development environments (see docker-compose file) and my hosting website.

Closes #1886

### CODE REVIEW

File: core/installation/views/steps/admin_account_setup.php [hunk 1]
```diff
@@ -118,8 +118,8 @@
 				<div class="sixteen wide mobile twelve wide tablet ten wide computer column">
 					<div class="ui form">
 						<?php
-							create_field('text', $language['username'], 'username', 'inputUsername');
-							create_field('email', $language['email_address'], 'email', 'inputEmail');
+							create_field('text', $language['username'], 'username', 'inputUsername', getenv('NAMELESS_ADMIN_USERNAME') ?: '');
```
Comment (samerton) [inline core/installation/views/steps/admin_account_setup.php:121]: Apparently `getenv` returns false if the variable isn't set - do you know if this results in the string "false" being added to the field without the environment variable?

Comment (Derkades) [reply core/installation/views/steps/admin_account_setup.php:121]: `getenv('NAMELESS_ADMIN_USERNAME') ?: ''` is short for `getenv('NAMELESS_ADMIN_USERNAME') ? getenv('NAMELESS_ADMIN_USERNAME') : '' `. I'm pretty sure the string `"false"` evaluates to the boolean `false` in php?

Comment (samerton) [reply core/installation/views/steps/admin_account_setup.php:121]: Sorry I didn't see `?:`, oops 😄

**Comment by Derkades** [comment]
"documentation" here: https://github.com/NamelessMC/Nameless/wiki/Environment-variables

**Pull request merged**